### ISSUE TITLE
Skidded my 0.30-WebGL Project 💀 

### ISSUE THREAD

**Issue by JessePinkman27** [Klockcraft-Official/.github#1]
You copied my ENTIRE 0.30 WebGL (Minecraft Classic WebGL) project WITHOUT giving me ANY credit, If you're gonna steal my project at least change the folder & package names and remove my name from the code 💀.

-JessePinkman27 (PeytonPlayz585)

**Comment by wxnnvs** [comment]
L nineyear olds

**Comment by Vexb0t** [comment]
> > You copied my ENTIRE 0.30 WebGL (Minecraft Classic WebGL) project WITHOUT giving me ANY credit, If you're gonna steal my project at least change the folder & package names and remove my name from the code 💀.
> > -JessePinkman27 (PeytonPlayz585)
> 
> DMCA em lol

bro is NOT mojang 🗣️🔥

**Comment by vukle** [comment]
> > You copied my ENTIRE 0.30 WebGL (Minecraft Classic WebGL) project WITHOUT giving me ANY credit, If you're gonna steal my project at least change the folder & package names and remove my name from the code 💀.
> > -JessePinkman27 (PeytonPlayz585)
> 
> DMCA em lol

mit license:

**Comment by wxnnvs** [comment]
> > > You copied my ENTIRE 0.30 WebGL (Minecraft Classic WebGL) project WITHOUT giving me ANY credit, If you're gonna steal my project at least change the folder & package names and remove my name from the code 💀.
> > > -JessePinkman27 (PeytonPlayz585)
> > 
> > 
> > DMCA em lol
> 
> mit license:

ahh fuk

**Comment by OtterCodes102** [comment]
> > > You copied my ENTIRE 0.30 WebGL (Minecraft Classic WebGL) project WITHOUT giving me ANY credit, If you're gonna steal my project at least change the folder & package names and remove my name from the code 💀.
> > > -JessePinkman27 (PeytonPlayz585)
> > 
> > 
> > DMCA em lol
> 
> mit license:

![image](https://github.com/Klockcraft-Official/.github/assets/144633084/62b15084-18aa-44c5-b059-def9832eb5bb)

**Comment by JessePinkman27** [comment]
> > > You copied my ENTIRE 0.30 WebGL (Minecraft Classic WebGL) project WITHOUT giving me ANY credit, If you're gonna steal my project at least change the folder & package names and remove my name from the code 💀.
> > > -JessePinkman27 (PeytonPlayz585)
> > 
> > 
> > DMCA em lol
> 
> mit license:

Yes, they are allowed to use it, BUT they are supposed to give me credit, which they didn't. 
They tried to steal it. Any person who looked at this would not know it was my project unless they were looking around in the code.

**Comment by wxnnvs** [comment]
> > > > You copied my ENTIRE 0.30 WebGL (Minecraft Classic WebGL) project WITHOUT giving me ANY credit, If you're gonna steal my project at least change the folder & package names and remove my name from the code 💀.
> > > > -JessePinkman27 (PeytonPlayz585)
> > > 
> > > 
> > > DMCA em lol
> > 
> > mit license:
> 
> Yes, they are allowed to use it, BUT they are supposed to give me credit, which they didn't. 
> They tried to steal it. Any person who looked at this would not know it was my project unless they were looking around in the code. Mojang could still DMCA the repo even tho it has an MIT licence because it still contains their "Intellectual Property"

Then you gotta go for it bro

**Comment by denied2023** [comment]
mr. a

**Comment by HudsonDukette** [comment]
just give him fucking credit

**Comment by firstlynx-sys** [comment]
dmca or give credit

**Comment by OtterCodes101** [comment]
bro just deleted the repo with the skidded code 💀

**Comment by denied2023** [comment]
i like boys

**Comment by vukle** [comment]
bro think he unique
